models.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):

```

Comment:
Rename it to `Author`. We don't use `DBAuthor` to indicate it's a model; in FastAPI, we differentiate using `model.Author`, `schemas.Author`, etc.